volestipy/setup.py

```diff
@@ -1,75 +1,83 @@
-from os.path import join
-from setuptools import setup
+# from setuptools import setup
+# from setuptools import Extension
+from distutils.core import setup
+from distutils.core import Extension
 from Cython.Build import cythonize
-from setuptools.extension import Extension
+from os.path import join
 import numpy
 
 
-version = "0.2.0"
+version = "0.3.0"
 license='LGPL3',
 packages = ["volestipy"]
 description="volestipy: wrapper for the VolEsti library to sample from convex sets and compute volume."
-author = "Pedro Zuidberg Dos Martires"
-author_email="pedro.zuidbergdosmartires@cs.kuleuven.be"
+author = "Pedro Zuidberg Dos Martires, Haris Zafeiropoulos"
+author_email="pedro.zuidbergdosmartires@cs.kuleuven.be, haris-zaf@hcmr.gr"
```

Comment:
I think it is ok for now, but also read this https://bugs.python.org/msg93284

Reply:
Ok! Thanks for sharing this!